packages/fx-core/resource/package.nls.json

```diff
@@ -198,7 +198,7 @@
   "plugins.spfx.cannotFindManifest": "Cannot find manifest file %s",
   "plugins.spfx.cannotGetSPOToken": "Cannot get SPO access token",
   "plugins.spfx.cannotGetGraphToken": "Cannot get Graph access token",
-  "plugins.spfx.insufficientPermission": "You don't have permission to upload and deploy package to App Catalog %s, use site admin account.",
+  "plugins.spfx.insufficientPermission": "You don't have permission to upload and deploy package to App Catalog %s, use site admin account, or you can get your own free Microsoft 365 tenant from [Microsoft 365 developer program](%s)",
```

Comment:
Align the string with initial error notification window

Reply:
Is this better: "Access denied. You are not authorized to upload and deploy packages to App Catalog %s. You can use a site admin account to complete the task. Alternatively, you can obtain your own free Microsoft 365 tenant through the [Microsoft 365 developer program](%s) and manage your own App Catalog."

Reply:
Both looks OK to me. @sffamily Any comments?

Reply:
Unable to upload and deploy package to App Catalog %s. You need your org's Microsoft 365 tenant admin permissions. You can get free Microsoft 365 tenant from [Microsoft 365 developer program](%s) for testing purpose.

Reply:
updated